src/Storage/StorageInterface.php

```diff
@@ -23,6 +23,7 @@ public function upload(object $obj, PropertyMapping $mapping): void;
      *
      * @param object          $obj     The object
      * @param PropertyMapping $mapping The mapping representing the field to remove
+     * @throw \Exception      Throws an exception
```

Comment:
An exception throwing in FileSystemStorage is still missing

Reply:
I tried to not change the behaviour as it is today. That's why I didn't add any.

Reply:
> I tried to not change the behaviour as it is today. That's why I didn't add any.

It should be caught too, and dispatch the new event.